### DISCUSSION TITLE
2022-06-08 Weekly Meeting 59 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#241] [Weekly Meetings]
Weekly Meeting 59 is happening on Wednesday June 8th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 
Lean Coffee and discuss [Privacy: Teleport, Ghost or Mute on Arrival · Issue #238 · omigroup...](https://github.com/omigroup/omigroup/issues/238)

Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - Second OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

**Comment by mrmetaverse** [comment]
Text chat here -> https://discord.com/channels/770382203782692945/839486903178362920/0

**Comment by mrmetaverse** [comment]
# Website group 
- going to work on getting the first blog posted

**Comment by mrmetaverse** [comment]
# Scripting 
- thirdroom creating gltf scene graph api that you can expose. Could be good for an interoperable scripting API 
  - we have been leaning towards something based around a gLTF format
  - working on finalizing an initial prototype
  - exploring how this might work inside of webassembly WASM (getting data in and out of a WASM context) 
- great discussion 
- Next week is the third developer hour open chat Tuesday at16:00 UTC  (9a PT / 12 ET)

**Comment by mrmetaverse** [comment]
# gLTF extensions

**Reply by robertlong** [reply]
Notes from the last meeting: https://github.com/omigroup/gltf-extensions/discussions/87

Highlights are that we needed to update the KHR_audio spec and @antpb did a lot of that work. We also spoke more about the OMI_ref extension which lets you embed glTF documents inside other glTF documents. Take a look if you're interested https://github.com/omigroup/gltf-extensions/pull/85

**Comment by mrmetaverse** [comment]
# Media 
- meeting next week 
- working on low-hanging fruit / small items in interim 
  - [twitter bot](https://github.com/omigroup/media/issues/40): Puts any post into the server so we can swarm with likes, retweets, and engage.

**Reply by mrmetaverse** [reply]
case study getting developed now about the 1 hour play from MetaTraversal

**Reply by mrmetaverse** [reply]
XRGuild is a NPO for all people working in XR. Focuses on educational support, best practices, ethics, and making sure we do right by each other. Similar to producers, guild writers, etc. 

Thoughts of a metacamp in discussion. This was something that came up here before that OMI can't do alone, so maybe this is a chance for us to collab.

**Reply by mrmetaverse** [reply]
There is great interest in talking about the UX of portals

**Reply by mrmetaverse** [reply]
A. proposed that we consider having more events on twitter spaces

**Reply by mrmetaverse** [reply]
We are a small media team, but if you have interesting endeavours we should explore, keep them coming, we just need to find people and time.

**Reply by mrmetaverse** [reply]
>BTW, we won't have time to talk about LAMINA1 but our colleague Tony Parisi announced today his work with Neil Stephenson to create a Layer-1 chain for the open metaverse (you may want to join the LAMINA1 Discord for those updates).

**Comment by mrmetaverse** [comment]
# New opportunities to form working groups: 
Interested in leading a new working group in OMI? Want to explore new ways to apply your skills? We have two recent proposed working groups worth reviewing. Just add your name in the comments of either one if you're interested. 

## UX Research 
- https://github.com/omigroup/omigroup/issues/222
## Portals 
- https://github.com/omigroup/omigroup/issues/209

**Comment by mrmetaverse** [comment]
# New concept proposal (privacy teleport) 
We reviewed Dr. Kim's recent submission: https://github.com/omigroup/omigroup/issues/238

**Reply by mrmetaverse** [reply]
This would also be perfect for a potential #209 working group

**Comment by mrmetaverse** [comment]
# Croquet Demo! ([Lean Coffee](https://www.figma.com/file/wxw2Jxe1bwmZTExfYgtLeY/OMI-lean-coffee-june-8?node-id=0%3A1) replacement)
join me here https://croquet.io/microverse/?q=81wq3jlrdc#pw=sP9Zb7UH-XKOqEZbHrR7zg

**Reply by mrmetaverse** [reply]
This would also be perfect for a potential https://github.com/omigroup/omigroup/issues/209 working group

**Reply by mrmetaverse** [reply]
![image](https://user-images.githubusercontent.com/63426722/172673589-19267048-08e2-42cf-8d16-1a80d5d78380.png)

**Reply by mrmetaverse** [reply]
![image](https://user-images.githubusercontent.com/63426722/172674435-fa7b4c94-bfc6-4bd1-a95a-c477de3f212a.png)

**Reply by mrmetaverse** [reply]
![image](https://user-images.githubusercontent.com/63426722/172675013-b9f3f6d8-54c8-49d0-95ac-348b3e6c968f.png)

**Reply by mrmetaverse** [reply]
similar ideas here to exokit
https://github.com/exokitxr/exokit-web

exokit was interesting in comparison in that it supported vr -- not sure if stock iframes could support stereo rendering just yet

https://hackmd.io/@exokit/exokit-web-v2
